api/src/wfl/boot.clj

```diff
@@ -87,6 +88,20 @@
                (.renameTo in-zip (io/file out-zip)))
              (finally (util/delete-tree directory)))))))
 
+(defn cromwellify-warp-wdl
+  "Cromwellify the WDL from warp in CLONES to RESOURCES."
+  [clones resources {:keys [release top] :as _wdl}]
+  (let [dp (str/join "/" [clones "warp"])]
+    (util/shell-io! "git" "-C" dp "checkout" release)
+    (let [[directory in-wdl in-zip] (wdl/cromwellify (io/file dp top))]
+      (when directory
+        (try (let [out-wdl (.getPath (io/file resources (.getName in-wdl)))
+                   out-zip (str (util/unsuffix out-wdl ".wdl") ".zip")]
+               (io/make-parents out-zip)
+               (.renameTo in-wdl (io/file out-wdl))
+               (.renameTo in-zip (io/file out-zip)))
+             (finally (util/delete-tree directory)))))))
+
```

Comment:
To be frank I have absolutely zero issue with this function basically being copied from the dsde-pipelines one. An abstracted version would require a rework when we got rid of dsde-pipelines anyway.

Reply:
Thanks, what you mentioned is the main reason I decided to not bother copy-pasting this function : P